sdk/keyvault/azkeys/CHANGELOG.md

```diff
@@ -3,12 +3,15 @@
 ## 0.3.0 (Unreleased)
 
 ### Features Added
-* Adds the `ReleasePolicy` parameter to the `UpdateKeyPropertiesOptions` struct
+* Adds the `ReleasePolicy` parameter to the `UpdateKeyPropertiesOptions` struct.
 * Adds the `Immutable` boolean to the `KeyReleasePolicy` model.
 
 ### Breaking Changes
 * Changed the `Data` to `EncodedPolicy` on the `KeyReleasePolicy` struct.
-* Changed the `Tags` properties from `map[string]*string` to `map[string]string`
+* Changed the `Tags` properties from `map[string]*string` to `map[string]string`.
+* Changed the `Updated`, `Created`, and `Expires` properties to `UpdatedOn`, `CreatedOn`, and `ExpiresOn`.
+* Renamed `JSONWebKeyOperation` to `KeyOperations`.
```

Comment:
Just singular i.e., `KeyOperation`: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyOperation.cs